src/backends/firebird/standard-into-type.cpp

```diff
@@ -119,11 +121,46 @@ void firebird_standard_into_type_backend::exchangeData()
                 blob->assign(*reinterpret_cast<ISC_QUAD*>(buf_));
             }
             break;
+
+        case x_longstring:
+            {
+                long_string* const dst = static_cast<long_string*>(data_);
+
```

Comment:
Do we need those empty lines?

It's matter of taste and perhaps belongs to the latest style and formatting and `clang-format` discussion.
I just find SOCI code a bit (too) sparse :-)

Reply:
I mostly try to emulate the existing style and the `case x_blob` just above does use them.

But, to be honest, I prefer too many blank lines to not enough of them, personally...

Reply:
Sure